.github/workflows/scorecard.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           persist-credentials: false
 
       - name: "Run analysis"
-        uses: ossf/scorecard-action@e38b1902ae4f44df626f11ba0734b14fb91f8f86 # v2.1.2
+        uses: ossf/scorecard-action@08b4669551908b1024bb425080c797723083c031 # v2.2.0
```

Comment:
I verified this is the hash for the v2.2.0 release:
https://github.com/ossf/scorecard-action/commit/08b4669551908b1024bb425080c797723083c031